x-pack/plugins/security_solution/server/lib/detection_engine/signals/build_bulk_body.ts

```diff
@@ -71,6 +71,7 @@ export const buildBulkBody = ({
     ...buildSignal([doc], rule),
     ...additionalSignalFields(doc),
   };
+  delete doc._source.threshold_result;
```

Comment:
Does this merit a comment about preventing the "meta signals pollution"? Or should we make this more implicit by moving this into a simple filtering function, for now?

Reply:
This is really about persisting values that belong to the signal before the the `signal` object has been created yet. I like the idea of using a filter, or even maybe architecting this a little differently so that the signal fields can be merged into the `signal` object, rather than having this function create the `signal` object. Will look at this in a follow-up PR.